dev_guide/builds.adoc

```diff
@@ -11,11 +11,76 @@ toc::[]
 
 == Overview
 A link:../architecture/core_objects/builds.html[build] is a process of creating
-runnable images to be used on OpenShift. There are three types of builds:
+runnable images to be used on OpenShift. There are three build strategies:
 
-- Docker build
-- STI build
-- custom build
+- link:../architecture/core_objects/builds.html#sti-build[Source-To-Image]
+- link:../architecture/core_objects/builds.html#docker-build[Docker]
+- link:../architecture/core_objects/builds.html#custom-build[Custom]
+
+== Defining a buildConfig
+
+A `*buildConfig*` describes a single build definition and a
```

Comment:
Why make a name reference to the REST endpoint when it is not even present in the document?!